mesa/visualization/solara_viz.py

```diff
@@ -102,24 +102,33 @@ def SolaraViz(
 
     # set up reactive model_parameters shared by ModelCreator and ModelController
     reactive_model_parameters = solara.use_reactive({})
+    reactive_play_interval = solara.use_reactive(play_interval)
 
     with solara.AppBar():
         solara.AppBarTitle(name if name else model.value.__class__.__name__)
 
     with solara.Sidebar(), solara.Column():
         with solara.Card("Controls"):
+            solara.SliderInt(
+                label="Play Interval (ms)",
+                value=reactive_play_interval.value,
+                on_value=lambda v: reactive_play_interval.set(v),
```

Comment:
I think you can replace the 2 value lines above with just `value=reactive_play_interval` . Since its a reactive variable it will automagically extract the value and update

Reply:
Thank you @Corvince! I checked the SliderInt documentation and found that it accepts both `int` and `solara.Reactive[int]` values:
```python
SliderInt(
    value: Union[int, solara.Reactive[int]] = 0,
    on_value: Optional[Callable[[int], None]] = None,
):
```
Based on this, I've removed the .value for better reactivity and cleaner code.